catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/image/NitfPreStoragePlugin.java

```diff
@@ -111,48 +115,51 @@ private boolean isNitfMimeType(String rawMimeType) {
 
     private void process(List<ContentItem> contentItems) {
         List<ContentItem> newContentItems = new LinkedList<>();
```

Comment:
Nit: it's pretty unusual to use a java `LinkedList` and not an `ArrayList`. There are cases for it, of course. Is there a performance benefit expected from using it here?